packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/TradeActions.tsx

```diff
@@ -0,0 +1,97 @@
+import { Row, variables } from '@trezor/components';
+import { hasBitcoinOnlyFirmware } from '@trezor/device-utils';
+import { analytics, EventType } from '@trezor/suite-analytics';
+import { goto } from 'src/actions/suite/routerActions';
+import { AppNavigationTooltip } from 'src/components/suite/AppNavigation/AppNavigationTooltip';
+import { Translation } from 'src/components/suite/Translation';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import styled, { css } from 'styled-components';
+import { HeaderActionButton } from 'src/components/suite/layouts/SuiteLayout/PageHeader/HeaderActionButton';
+import { selectDevice } from '@suite-common/wallet-core';
+import { spacings } from '@trezor/theme';
+import { SelectedAccountStatus } from '@suite-common/wallet-types';
+
+// instant without computing the layout
+const ShowOnLargeDesktopWrapper = styled.div<{ $isActive?: boolean }>`
+    ${({ $isActive }) =>
+        $isActive &&
+        css`
+            ${variables.SCREEN_QUERY.BELOW_DESKTOP} {
+                display: none;
+            }
+        `}
+`;
+interface TradeActionsProps {
+    analyticsEventType: EventType.AccountsActions | EventType.MenuActions;
+    selectedAccount?: SelectedAccountStatus;
+    hideBuyAndSellBelowDesktop?: boolean;
+}
+
+export const TradeActions = ({
+    analyticsEventType,
```

Comment:
can we decide what to tracked based on current route please?

Reply:
fixed